components/brave_new_tab_ui/containers/newTab/index.tsx

```diff
@@ -502,6 +512,15 @@ class NewTabPage extends React.Component<Props, State> {
     this.props.actions.setUserTLDAutoSet()
   }
 
+  onBraveTodayInteracting = (isInteracting: boolean) => {
+    // TODO(petemill): Detect percentage of scroll towards brave today
+    // and set opacity at that level, instead of a binary on/off.
```

Comment:
that would be cool! 😻 is it possible to do gradients with opacity?

Reply:
Ah this is already done, just not pushed. Not sure if I like how it feels actually, but will see.